packages/experience-builder-sdk/package.json

```diff
@@ -57,9 +62,13 @@
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
     "jsdom": "^21.1.0",
+    "postcss-import": "^15.1.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "relative-deps": "^1.0.7",
```

Comment:
was this dependency left on accident? (is also appears in the `prepare` command)